minio/go/src/cloud/benchflow/commons/minio/minio.go

```
@@ -53,11 +31,29 @@ func callMinioClient(fileName string, minioHost string, minioKey string) {
 		fmt.Println("Result: " + out.String())
 }
 
-func GenerateKey(fileName string) string {
-	// TODO: Use the hash library to generate a hash
-	trialId := os.Getenv("TRIAL_ID")
-	t := strings.Split(trialId, "_")
-	return ("hash/"+t[0]+"/"+t[1]+"/"+os.Getenv("CONTAINER_NAME")+"/"+os.Getenv("COLLECTOR_NAME")+"/"+os.Getenv("DATA_NAME")+"/"+fileName)
+func sendToMinio(fileName string, minioHost string, minioKey string, accessKey string, secretAccessKey string) {
```

You might want to return the error to the caller, so it can decide what to do